Spotify UI/lib/helpers/size.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+getHeightOfDevice(context) => MediaQuery.of(context).size.height;
+getWidthOfDevice(context) => MediaQuery.of(context).size.width;
```

Comment:
@cccarle These two lines could be refactored using dart extension functions on BuildContext. It would make for an even cleaner code.